src/UnityExtension/Assets/Editor/GitHub.Unity/UI/HistoryView.cs

```diff
@@ -321,8 +328,19 @@ public void OnEmbeddedGUI()
 
                 GUILayout.FlexibleSpace();
 
+                GUI.enabled = currentRemote != null;
+                var fetchClicked = GUILayout.Button(FetchButtonText, Styles.HistoryToolbarButtonStyle);
+                GUI.enabled = true;
+                if (fetchClicked &&
+                    EditorUtility.DisplayDialog(FetchConfirmTitle,
```

Comment:
We probably don't need to ask for confirmation, fetch is a non-destructive operation, we can just do it.